package/MDAnalysis/analysis/density.py

```diff
@@ -408,8 +417,15 @@ def _prepare(self):
             # ideal solution would use images: implement 'looking across the
             # periodic boundaries' but that gets complicated when the box
             # rotates due to RMS fitting.
-            smin = np.min(coord, axis=0) - self._padding
-            smax = np.max(coord, axis=0) + self._padding
+            try:
+                smin = np.min(coord, axis=0)
+                smax = np.max(coord, axis=0)
+            except ValueError as err:
+                errmsg = ("No atoms in selection at current frame"
```

Comment:
This needs to be a bit more explicit. Think about what users would understand if faced with this error.

Couple of pointers:
1) "selection" is ambiguous, would AtomGroup be more appropriate
2) "no user defined grid" -> ok, but what should users be doing?

Reply:
"please enter grid values" - I'll add this